tests/test_responder.py

```diff
@@ -815,9 +814,9 @@ def get(req, resp):
 
 def create_asset(static_dir: Path, name=None, parent_dir: str = None) -> Path:
     if name is None:
-        name = random.choices(string.ascii_letters, k=6)  # noqa: S311
+        name = "".join(random.choices(string.ascii_letters, k=6))  # noqa: S311
         # :3
-        ext = random.choices(string.ascii_letters, k=2)  # noqa: S311
+        ext = "".join(random.choices(string.ascii_letters, k=2))  # noqa: S311
         name = f"{name}.{ext}"
```

Comment:
Make those random characters actual strings by joining the characters together, otherwise they will serialize into Python's `list` representation.